### PR TITLE
OP-1130 improve JPA methods - dicom package

### PR DESCRIPTION
**Pull request by dbmalkovsky**
See OP-1130

Changes for `dicom` package

Fixed some spelling issues too

Note there are no changes in the `API` repo

### CODE REVIEW

File: src/main/java/org/isf/dicom/service/DicomIoOperations.java [hunk 1]
```diff
@@ -48,14 +48,14 @@ public class DicomIoOperations
 	private DicomIoOperationRepository repository;
 	
 	/**
-	 * Load a list of id file for series
+	 * Load a list of file iss for a series.
```
Comment (mwithi) [inline src/main/java/org/isf/dicom/service/DicomIoOperations.java:51]: Like this?
```suggestion
	 * Load a list of file id for a series.
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

File: src/main/java/org/isf/dicom/service/DicomIoOperations.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/dicom/service/DicomIoOperations.java:51]: yes and adding the `s` to `serie`;   `series` is both singular and plural :smile:

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**